oneclick_monolith_install/install-config.yml

```diff
@@ -0,0 +1,18 @@
+# Name to uniquely identify your CommCareHQ instance/environment.
+env_name: ""
+
+# Web address of your HQ instance
+site_host: ""
+
+# Ansible inventory name for this server (e.g. hqserver1)
+server_inventory_name: ""
```

Comment:
is this necessary? we could just default this to `commcare` or the `env_name` from above?

Reply:
My reasoning to offer this in the config was this.

This is a name that will be used/visible frequently when referring the host via ansible or in the sentry/datadog logs, and in the email alerts etc. Also, updating this in the future takes more effort too (needs ansible redeploy).  So, I thought better to have the user pick a name.

Reply:
That's fair, although I think if this is the only machine the name here is somewhat irrelevant. If they end up moving to a cluster type deployment, they will need to redeploy ansible anyway.